resources/zib/zib-MedicationAgreement.xml

```diff
@@ -0,0 +1,269 @@
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationAgreement.AdditionalInformation" />
+      </type>
+    </element>
+    <element id="MedicationRequest.extension:instructionsForUseDescription">
+      <path value="MedicationRequest.extension" />
+      <sliceName value="instructionsForUseDescription" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-InstructionsForUse.Description" />
+      </type>
+    </element>
+    <element id="MedicationRequest.extension:periodOfUse">
+      <path value="MedicationRequest.extension" />
+      <sliceName value="periodOfUse" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-PeriodOfUse" />
+      </type>
+    </element>
+    <element id="MedicationRequest.extension:periodOfUse.value[x]">
+      <path value="MedicationRequest.extension.value[x]" />
+      <definition value="**Start date**: This is the time at which the agreement was to take effect (or took effect or will take effect). This is the time at which the instructions for use in this agreement start. In the case of an agreement to discontinue use, this is the start date of the original medication agreement. The end date indicates from when the medication is to be discontinued.&#xD;&#xA;&#xD;&#xA;**End date**: The time at which the period of use ends (or ended or will end). In the case of an agreement to discontinue use, this is the time at which the medication is to be discontinued. To avoid confusion between 'to' and 'up to', the submission of time is always mandatory for the end date.&#xD;&#xA;&#xD;&#xA;With medication for an indefinite period only a start date is indicated." />
+    </element>
+    <element id="MedicationRequest.extension:periodOfUseDuration">
+      <path value="MedicationRequest.extension" />
+      <sliceName value="periodOfUseDuration" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-PeriodOfUse-Duration" />
+      </type>
+    </element>
+    <element id="MedicationRequest.extension:periodOfUseDuration.value[x]">
+      <path value="MedicationRequest.extension.value[x]" />
+      <definition value="**Duration**: The intended duration of use. E.g. 5 days or 8 weeks. It is not allowed to indicate the duration in months, because different months have a variable duration in days." />
+    </element>
+    <element id="MedicationRequest.modifierExtension">
+      <path value="MedicationRequest.modifierExtension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationRequest.modifierExtension:stopType">

```

Comment:
The two concept from the ValueSet can be perfectly mapped to the required FHIR element `.status`. Wouldn't this be better than to include another extension?

Reply:
I really have a hard time understanding this reasoning. See #109

Reply:
Created #114 and organized a meeting with Arianne/Marijke to discuss this. The proposed way forward is to go with the current design, consistent with STU3, and potentially adjust this later in the profiles and mappings.